allium/lib/categorization.py

```
@@ -653,13 +653,98 @@ def propagate_as_rarity(relay_set):
         cw = e.get('consensus_weight_fraction', 0)
         relay['as_cw_label'] = f"{cw * 100:.2f}%" if cw >= 0.0005 else ("<0.05%" if cw > 0 else "0%")
 
+def _build_incomplete_aroi_sibling_map(relay_set):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_build_incomplete_aroi_sibling_map`

(ANN202)